projects/igniteui-angular/src/lib/chips/chip.component.ts

```diff
@@ -528,3 +530,9 @@ export class IgxChipComponent implements AfterViewInit {
     }
     // End chip igxDrop behaviour
 }
+
+export interface ChipSelectEventArgs {
+    owner: IgxChipComponent;
+    cancel: boolean;
+    nextStatus: boolean;
```

Comment:
Could you rename `nextStatus` to `selected` since the second one is probably easier to understand.